jenkins.py

```diff
@@ -143,7 +144,7 @@ def run():
   else:
     which = 'which'
 
-  polly_root = os.getenv('POLLY_ROOT')
+  polly_root = os.path.abspath(os.getenv('POLLY_ROOT'))
```

Comment:
Should be checked for `None`:
- https://github.com/ruslo/hunter/commit/9e43e66c172663c5f4234549be75c9a6c9255bf4

See:
- https://travis-ci.org/ingenue/hunter/jobs/531915099

Reply:
Thanks!